pep-0604.rst

```diff
@@ -123,7 +169,7 @@ For more details about discussions, see links below:
 
```

Comment:
The headline just above here says `Union[type1|type2]`. I think that `|` is a mistake and the intention was to write `Union[type1, type2]`.